frontend/src/bundles/common/components/select/select.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+    FormControl,
+    FormErrorMessage,
+    FormLabel,
+    Select as LibrarySelect,
+} from '@chakra-ui/react';
+import {
+    type FieldInputProps,
+    type FormikValues as FormValues,
+    Field,
+} from 'formik';
+
+import { useFormField } from '~/bundles/common/hooks/hooks.js';
+
+type Properties<T extends FormValues> = {
+    label: string;
+    name: FieldInputProps<T>['name'];
+    children: React.ReactNode;
+    required?: boolean;
+    placeholder?: string;
+};
+
+const Select = <T extends FormValues>({
+    label,
+    name,
+    children,
+    required = false,
+    placeholder = '',
+}: Properties<T>): JSX.Element => {
```

Comment:
```suggestion
}: Properties<T>):  React.FC<Properties<T>>=> {
```
Maybe update to react FC?